src/coreclr/tools/Common/TypeSystem/Interop/IL/Marshaller.cs

```diff
@@ -1581,6 +1581,10 @@ protected override void EmitCleanupManaged(ILCodeStream codeStream)
 
     class AnsiStringMarshaller : Marshaller
     {
+        const int MAX_LOCAL_BUFFER_LENGTH = 256; // TODO: Is this accurate on all platforms?
+
+        private ILLocalVariable m_localBuffer = default;
```

Comment:
```suggestion
        private ILLocalVariable _localBuffer = default;
```
Our convention for C# is to use `_` prefix for instance fields.

Reply:
yup will rename, once current CI job completes.